### ISSUE TITLE
Delay Compensator Mode button not working as described

### ISSUE THREAD

**Issue by markk** [sadko4u/lsp-plugins#142]
The documentation [here](https://lsp-plug.in/?page=manuals&section=comp_delay_stereo) states:

```
Mode - the switch button that selects active mode of the delay: Samples, Distance or Time. 
When the mode is active, then the Active led of the appropriate control group is shining.
```
Instead, clicking on the button cycles between three states:

1. All three Active leds off (small, dull green), and calculated values shown in the monitoring section.
2. All three Active leds on (larger, bright green), all values shown as 0 in the monitoring section.
3. No change from state 2.

Only in state 1 is the audio signal delayed.

Whilst it would be good to have the button working as documented, I wonder if it might be better for the settings just to be linked so that adjusting the distance automatically updates the samples and time sections? Then the mode switch would be unnecessary.

(Additionally, it might be more consistent to have the values in the monitoring section listed in the same order as the sections appear, i.e. Samples, Distance, Time.)

Release 1.1.26

**Comment by sadko4u** [comment]
Yes, that seems to be a regression. Thanks for reporting.

**Comment by sadko4u** [comment]
The regression is in Led indication. The button that switches modes works properly. Pushed changes to **github-issue-142** branch. Are you available to verify?

**Comment by markk** [comment]
Sorry for the delay - yes this is now working perfectly. Thanks!

**Comment by sadko4u** [comment]
Please don't close tickets until I publish the release

**Comment by sadko4u** [comment]
Merged changes into **devel** branch: 706811f111887481a4926b120a73c8dbdd95b7a9

**Comment by sadko4u** [comment]
Removed **github-issue-142** branch.

**Comment by sadko4u** [comment]
Available in 1.1.28 release!
